configs/solo/solov2_x101_dcn_fpn_3x_coco.py

```diff
@@ -0,0 +1,121 @@
+_base_ = [
```

Comment:
add configs/solo/solov2_r50_fpn_1x_coco.py to _base_ and remove duplicate parameters

Reply:
@jshilong do we need to add new detector named `SOLOv2 `?

Reply:
@jshilong great suggestion. I will change it.

Reply:
@BIGWangYuDong Is it really necessary to add one more pure alias class?